src/Nancy/Json/Simple/SimpleJson.cs

```diff
@@ -1787,6 +1787,15 @@ public static IEnumerable<PropertyInfo> GetProperties(Type type)
 #endif
         }
 
+        public static PropertyInfo GetProperty(Type type, string name)

```

Comment:
As it turns out, throwing a `.GetTypeInfo()` on the call site removed the need for this. Reverting.